README.md

```diff
@@ -72,23 +74,21 @@ let package = Package(
 
 Then run `swift build` to download, compile, and link all your dependencies.
 
-### OR: Install the Driver Using CocoaPods
-*Please make sure you have followed the instructions in the previous section on installing the MongoDB C Driver before proceeding.*
-
-CocoaPods is a dependency manager for Swift and Objective-C. You can install it by running `gem install cocoapods`. See [the CocoaPods documentation](https://cocoapods.org/) for more information.
+## iOS, tvOS, and watchOS
+Installation is supported via [CocoaPods](https://cocoapods.org/). The pod includes as a dependency an embedded version of the MongoDB C Driver, meant for use on these OSes. 
```

Comment:
Should we mention that this build currently does not support SSL (https://github.com/mongodb/mongo-swift-driver/issues/141)?

Reply:
Yup, I think this is an important thing to mention in the README.

Reply:
done.